### PR TITLE
Explicitly annotate angularjs injection

### PR DESCRIPTION
**Pull request by christopherthielen**
Make sure to turn whitespace diffing off in the github UI, because prettier did a lot of reformatting: https://github.com/spinnaker/deck/pull/6587/files?utf8=%E2%9C%93&diff=split&w=1

I propose we enforce explicit AngularJS DI annotation instead of using a) `ngInject`and loaders or b) inferring DI from the function arguments.  This eslint  should ensure that we can minify all our bundles.  As a side effect, this will also resolve the Safari issue we have observed where angular tries to inject jibberish.

I wrote an eslint rule with a fixer that enforces explicit AngularJS injection, it looks like this:

eslint angularjs rule

The eslint rule will come in a subsequent PR because it requires updating of eslint and adding eslint-typescript packages.

### CODE REVIEW

**Comment by christopherthielen** [comment]
conflicts ... this is going to be painful to rebase 😢

File: app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts [hunk 1]
```diff
@@ -122,15 +122,20 @@ export class AwsServerGroupConfigurationService {
     'Default',
   ];
 
+  public static $inject = [
```
Comment (anotherchrisberry) [inline app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts]: Did we shut off the lint rule that complains about statics not being first?

Comment (anotherchrisberry) [reply app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts]: That was rhetorical, it's called `member-ordering` and we did.

Comment (christopherthielen) [reply app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts]: ye, finally.  what a horrible rule 4107b1d4f7fc1cfdefa24801a1b14ae5be565537

**Comment by christopherthielen** [comment]
nice thanks for looking @anotherchrisberry

**Pull request merged**